routemaster/config/schema.yaml

```diff
@@ -170,6 +170,14 @@ properties:
           required:
             - class
           additionalProperties: false
+  # A root element for placeholders which can be overriden within the layers of
+  # configuration files. Since these end up in the structure but are ignored,
+  # the schema needs to allow them to exist though cannot describe the children
+  # of the node in detail.
+  placeholders:
```

Comment:
Not completely sold on this name yet. Would definitely appreciate suggestions for something better.

`override` feels the most obvious alternative, however since you can also use this mechanism just within a single file (for the convenience of de-duplicating stuff) it seems less good than `placeholders` is.

Reply:
Some ideas:
 - `context`
 - `vars`
 - `variable{s}`
 - `lookup{s}`

Reply:
Small +1 to `variables` _maybe_? But `placeholders` feels fine?

Reply:
Yeah, let's go with this as-is.